config/accounts.go

```diff
@@ -45,13 +45,16 @@ type CookieSync struct {
 type AccountCCPA struct {
 	Enabled            *bool              `mapstructure:"enabled" json:"enabled,omitempty"`
 	IntegrationEnabled AccountIntegration `mapstructure:"integration_enabled" json:"integration_enabled"`
+	ChannelEnabled     AccountIntegration `mapstructure:"channel_enabled" json:"channel_enabled"`
 }
 
 // EnabledForIntegrationType indicates whether CCPA is turned on at the account level for the specified integration type
 // by using the integration type setting if defined or the general CCPA setting if defined; otherwise it returns nil
 func (a *AccountCCPA) EnabledForIntegrationType(integrationType IntegrationType) *bool {
 	if integrationEnabled := a.IntegrationEnabled.GetByIntegrationType(integrationType); integrationEnabled != nil {
 		return integrationEnabled
+	} else if channelEnabled := a.ChannelEnabled.GetByIntegrationType(integrationType); channelEnabled != nil {
+		return channelEnabled
 	}
```

Comment:
It looks like this is the `if-else if` block that will be altered in the follow up PR when we completely phase out support for `integration-enabled` in the account config.